tests/std/tests/P3107R5_enabled_specializations/test.compile.pass.cpp

```diff
@@ -122,11 +124,23 @@ static_assert(enable_nonlocking_formatter_optimization<nullptr_t>);
 static_assert(enable_nonlocking_formatter_optimization<void*>);
 static_assert(enable_nonlocking_formatter_optimization<const void*>);
 static_assert(enable_nonlocking_formatter_optimization<pair<int, int>>);
+static_assert(enable_nonlocking_formatter_optimization<pair<int&, int>>);
+static_assert(enable_nonlocking_formatter_optimization<pair<int, int&>>);
 static_assert(!enable_nonlocking_formatter_optimization<pair<unoptimized, int>>);

```

Comment:
This is testing only references, but as the issue mentions, it's using `remove_cvref_t` to also handle `const` scenarios, which we should test.